core/chain-configs/src/lib.rs

```diff
@@ -52,6 +52,9 @@ pub const BLOCK_PRODUCER_KICKOUT_THRESHOLD: u8 = 90;
 /// Criterion for kicking out chunk producers.
 pub const CHUNK_PRODUCER_KICKOUT_THRESHOLD: u8 = 90;
```

Comment:
should we also make them 80 given that the default is 80 in mainnet?

Reply:
I don't want to break existing logic. Moreover, there may be tests checking that **at genesis** threshold was at 90 and then it was lowered to 80...